e2e-test-server/endtoendserver/constants.go

```diff
@@ -32,16 +32,27 @@ const (
 )
 
 var (
-	subscriptionMode        = mustGetenv("SUBSCRIPTION_MODE")
-	projectID               = mustGetenv("PROJECT_ID")
-	requestSubscriptionName = mustGetenv("REQUEST_SUBSCRIPTION_NAME")
-	responseTopicName       = mustGetenv("RESPONSE_TOPIC_NAME")
+	subscriptionMode        string
+	projectID               string
+	requestSubscriptionName string
+	responseTopicName       string
 )
 
-func mustGetenv(key string) string {
-	v := os.Getenv(key)
-	if v == "" {
-		log.Fatalf("environment variable %q must be set", key)
+func init() {
+	subscriptionMode = os.Getenv("SUBSCRIPTION_MODE")
+	if subscriptionMode == "" {
+		log.Fatalf("environment variable SUBSCRIPTION_MODE must be set")
+	}
+	projectID = os.Getenv("PROJECT_ID")
+	if projectID == "" {
+		log.Fatalf("environment variable PROJECT_ID must be set")
+	}
+	requestSubscriptionName = os.Getenv("REQUEST_SUBSCRIPTION_NAME")
+	if requestSubscriptionName == "" {
+		log.Fatalf("environment variable REQUEST_SUBSCRIPTION_NAME must be set")
+	}
+	responseTopicName = os.Getenv("RESPONSE_TOPIC_NAME")
+	if responseTopicName == "" {
+		log.Fatalf("environment variable RESPONSE_TOPIC_NAME must be set")
```

Comment:
this doesn't look great, but I couldn't find another way around the deep-exit linter. It doesn't have any configuration that I can find, so I can't skip this package. I tried calling `mustGetenv` from `init()` hoping that would solve it, but that still failed.

Reply:
You can disable the linter per-line if you think it isn't worth it (e.g. for a test).  See this as an example: https://github.com/open-telemetry/opentelemetry-go/blob/aa868d506b7e4de2f534f47b168b27b7f8824351/sdk/metric/metricdata/temporality.go#L24